core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -262,15 +294,29 @@ var TopicsAndSkillsDashboardPage = function() {
   };
 
   this.filterSkillsByStatus = async function(status) {
+    let width = (await browser.manage().window().getSize()).width;
+    if (width < 831) {
+      await action.click('Skill Filter', openFilter);
+    }
+
     await action.click(
       'Skill Dashboard status filter', skillStatusFilterDropdown);
     var dropdownOption = element(
       by.cssContainingText('mat-option .mat-option-text', status));
     await action.click(
       'Skill status filter option: ' + status, dropdownOption);
+
+    if (width < 831) {
```

Comment:
Any specific reason for why you have chosen 831?

Reply:
At width < 831, the UI on the topic and skill dashboard switches to the mobile view.